mcs/class/System/System.Diagnostics/PerformanceCounterCategory.cs

```diff
@@ -283,7 +300,12 @@ public static bool InstanceExists (string instanceName, string categoryName, str
 			CheckCategory (categoryName);
 			if (machineName == null)
 				throw new ArgumentNullException ("machineName");
-			int val = InstanceExistsInternal (instanceName, categoryName, machineName);
+
+			//?FIXME: machine appears to be wrong
+			//if (!ValidMachine (machineName))
+			//return false;
+
+			int val = InstanceExistsInternal (instanceName, categoryName);
```

Comment:
could we make this return a bool instead of an int while we're at it?

Reply:
That'd be line damage but I might have partly fixed it in nearby PR.

Reply:
you're touching this code block anyway, it is only line damage when touching unrelated code :)

Reply:
It is all related.